wildfly@.service

```diff
@@ -22,7 +22,7 @@ ExecStart=/bin/bash -c 'docker run -P --name ${CONTAINER} --restart=always \
   --hostname %H \
   --env ADMIN_USERNAME=$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/wildfly/admin-username?raw) \
   --env ADMIN_PASSWORD=$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/wildfly/admin-password?raw) \
-  --env JAVA_OPTS="-Xms64m -Xmx8192m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true" \
+  --env JAVA_OPTS="-Xms64m -Xmx12g -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true" \
```

Comment:
This change is unrelated to adding NewRelic, so I'll move it out of this PR if you want. It's just giving the WildFlys more RAM based on additional experience with what needs RAM on the CoreOS servers.